test/message_bus/message_inbox/confirm_message.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Refer this comment for proof data placeholder
https://github.com/mosaicdao/mosaic-1/pull/163#discussion_r377448271

Reply:
Put the proof data files in test/data